src/providers/spatialite/qgsspatialiteprovider.cpp

```diff
@@ -5045,7 +5045,7 @@ bool QgsSpatiaLiteProvider::checkLayerType()
         // Check if the whole sql is aliased i.e. '(SELECT * FROM \\"somedata\\" as my_alias\n)'
         if ( tableAlias.isEmpty() )
         {
-          regex.setPattern( QStringLiteral( R"re(\s+AS\s+(\w+)\n?\)?$)re" ) );
+          re.setPattern( QStringLiteral( R"re(\s+AS\s+(\w+)\n?\)?$)re" ) );
```

Comment:
This seems to be a mistake, the variable regex is unchanged on 4999.

Reply:
Na this is actually correcting a mistake